File: cpp/src/task.cpp
#include <string>
#include <iostream>
#include <fstream>

// WARNING: This code has nothing to do with the task at hand. Feel free to delete it.

int main(int argc, char** argv)
{
    std::ifstream ifs(argv[1]);
    std::ofstream ofs(argv[2]);
    std::string input;
    ifs >> input;
    ofs << "ANSWER" << input;
}

File: python/task/task.py
import sys

# WARNING: This code has nothing to do with the task at hand. Feel free to delete it.

if __name__ == '__main__':
    with open(sys.argv[1], "r") as i, open(sys.argv[2], "w") as o:
        value = i.read()
        o.write("ANSWER" + value)
